robottelo/constants/__init__.py

```diff
@@ -1,6 +1,15 @@
 """Defines various constants"""
 from nailgun import entities
 
+# String Color codes
+color_codes = {
```

Comment:
An enum would be appropriate here

Reply:
Or, even better, pull in colorama and use it for all of this.
https://pypi.org/project/colorama/

Reply:
Adding colorama would be more that needed for this simple script use! Let me make it enaum !

Reply:
BTW, Enum didnt work for me the best hence I used `Box` :)